pkg/authorization/api/casts.go

```diff
@@ -22,7 +22,6 @@ func ToPolicy(in *ClusterPolicy) *Policy {
 
 	ret := &Policy{}
 	ret.ObjectMeta = in.ObjectMeta
-	ret.Namespace = ""
```

Comment:
I pulled these out.  Decided it wasn't the responsibility of the cast to purify the object, merely pass it through.  It also made the validation unit tests capable of running negative tests.